chia/rpc/full_node_rpc_client.py

```diff
@@ -159,6 +159,12 @@ async def get_coin_records_by_parent_ids(
         response = await self.fetch("get_coin_records_by_parent_ids", d)
         return [CoinRecord.from_json_dict(coin_record_dict_backwards_compat(coin)) for coin in response["coin_records"]]
 
+    async def get_aggsig_additional_data(self) -> str:
+        result = await self.fetch("get_aggsig_additional_data", {})
+        add_data: str = result["additional_data"]
+        assert add_data is not None
```

Comment:
This is not necessary according to the hint above so one of them is wrong.  I don't think you need to assign this to a variable though, just return `hexstr_to_bytes` (or `bytes32.from_hexstr`) on `result["additional_data"]`.

Reply:
Done